src/sass/_base_component.scss

```diff
@@ -188,3 +188,24 @@ $accent-a-light: #c9f2f5;
   border: 3px solid #F0CC00;
   transform: rotate(102.02deg);
 }
+
+.large-line {
```

Comment:
Can we please rename to: `.large-slanted-line`? Same for medium-line and small-line class names.